dbt/include/clickhouse/macros/materializations/view.sql

```diff
@@ -67,11 +67,32 @@
 {%- endmaterialization -%}
 
 
+{% macro get_sql_security_clause(relation) %}
+  {% set sql_security = config.get('sql_security') %}
+  {% if sql_security -%}
+    {% if sql_security == 'definer' -%}
+    {%- set definer = config.require('definer') -%}
+      {% if not definer -%}
+        {{ exceptions.raise_compiler_error("Invalid config parameter `definer`. No value was provided.") }}
+      {%- endif %}
+      DEFINER = {{ definer }} SQL SECURITY DEFINER
+    {%- elif sql_security == 'invoker' %}
+      SQL SECURITY INVOKER
+    {%- elif sql_security == 'none' %}
```

Comment:
I think we better keep this option unimplemented, as it is deprecated

Reply:
@BentsiLeviav But only live view marked as deprecated, other functionality is not https://clickhouse.com/docs/en/sql-reference/statements/create/view#sql_security. Am I wrong?

Reply:
Just to be clear, I meant the none option, as the documentation indicates the following:

> SQL SECURITY NONE is a deprecated option. Any user with the rights to create views with SQL SECURITY NONE will be able to execute any arbitrary query. Thus, it is required to have GRANT ALLOW SQL SECURITY NONE TO <user> in order to create a view with this option.